streaming/java/streaming-api/src/main/java/io/ray/streaming/api/function/internal/RichFunctions.java

```diff
@@ -0,0 +1,37 @@
+package io.ray.streaming.api.function.internal;
+
+import io.ray.streaming.api.context.RuntimeContext;
+import io.ray.streaming.api.function.Function;
+import io.ray.streaming.api.function.RichFunction;
+
+public class RichFunctions {
```

Comment:
What is this class use for? Give some comments?

Reply:
It's a util class for `Function`, I think renaming it to `Functions` would be more readable.